include/revng/ADT/GenericGraph.h

```diff
@@ -1541,6 +1541,7 @@ template<StrictSpecializationOfBidirectionalNode T>
 struct GraphTraits<llvm::Undirected<T *>> {
 public:
   using NodeRef = T *;
+  using EdgeRef = typename T::Edge;
```

Comment:
Squash `GenericGraph: edge traits for llvm::Undirected` into the previous commit, no need to add the feature in a second step.

Reply:
Right, I squashed the mentioned commit into the previous one in a single step